chart/templates/deployment.yaml

```diff
@@ -143,6 +143,11 @@ spec:
           subPath: cacerts.pem
           readOnly: true
 {{- end }}
+{{- if and .Values.customLogos.enabled (or (eq .Values.customLogos.volumeKind "persistentVolumeClaim") (and (eq .Values.customLogos.volumeKind "configMap") (.Values.customLogos.volumeName))) }}
```

Comment:
Nit: You could make this a helper, in case someone modifies in the future and line 189 is forgotten.